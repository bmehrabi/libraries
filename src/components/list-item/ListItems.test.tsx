import React from 'react';
import {render} from '@testing-library/react';
import {BrowserRouter} from 'react-router-dom';
import {mainListItems} from './ListItems';

test('matches the snapshot', () => {
    const {container} = render(<BrowserRouter>{mainListItems}</BrowserRouter>);

    expect(container).toMatchSnapshot();
});
