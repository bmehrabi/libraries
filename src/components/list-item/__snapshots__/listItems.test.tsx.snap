// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<div>
  <a
    href="/"
  >
    <div
      class="MuiButtonBase-root MuiListItemButton-root MuiListItemButton-gutters MuiListItemButton-root MuiListItemButton-gutters css-16ac5r2-MuiButtonBase-root-MuiListItemButton-root"
      role="button"
      tabindex="0"
    >
      <div
        class="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="SearchIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14"
          />
        </svg>
      </div>
      <div
        class="MuiListItemText-root css-tlelie-MuiListItemText-root"
      >
        <span
          class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
        >
          Search
        </span>
      </div>
      <span
        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
      />
    </div>
  </a>
  <a
    href="/development-process"
  >
    <div
      class="MuiButtonBase-root MuiListItemButton-root MuiListItemButton-gutters MuiListItemButton-root MuiListItemButton-gutters css-16ac5r2-MuiButtonBase-root-MuiListItemButton-root"
      role="button"
      tabindex="0"
    >
      <div
        class="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="CodeIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M9.4 16.6 4.8 12l4.6-4.6L8 6l-6 6 6 6zm5.2 0 4.6-4.6-4.6-4.6L16 6l6 6-6 6z"
          />
        </svg>
      </div>
      <div
        class="MuiListItemText-root css-tlelie-MuiListItemText-root"
      >
        <span
          class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
        >
          Development process
        </span>
      </div>
      <span
        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
      />
    </div>
  </a>
  <a
    href="/about"
  >
    <div
      class="MuiButtonBase-root MuiListItemButton-root MuiListItemButton-gutters MuiListItemButton-root MuiListItemButton-gutters css-16ac5r2-MuiButtonBase-root-MuiListItemButton-root"
      role="button"
      tabindex="0"
    >
      <div
        class="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="InfoIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2m1 15h-2v-6h2zm0-8h-2V7h2z"
          />
        </svg>
      </div>
      <div
        class="MuiListItemText-root css-tlelie-MuiListItemText-root"
      >
        <span
          class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
        >
          About Me
        </span>
      </div>
      <span
        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
      />
    </div>
  </a>
</div>
`;
